src/java/org/apache/cassandra/cql3/statements/schema/AlterTableStatement.java

```diff
@@ -553,6 +557,41 @@ public void validate(ClientState state)
             validateDefaultTimeToLive(attrs.asNewTableParams());
         }
 
+        private TableParams validateAndUpdateTransactionalMigration(TableParams prev, TableParams next)
+        {
+            if (next.transactionalMode.accordIsEnabled && SchemaConstants.isSystemKeyspace(keyspaceName))
+                throw ire("Cannot enable accord on system tables (%s.%s)", keyspaceName, tableName);
+
+            boolean modeChange = prev.transactionalMode != next.transactionalMode;
+            boolean wasMigrating = prev.transactionalMigrationFrom.isMigrating();
+            boolean forceMigrationChange = prev.transactionalMigrationFrom != next.transactionalMigrationFrom;
+
+            if (modeChange && next.transactionalMode.accordIsEnabled && !DatabaseDescriptor.getAccordTransactionsEnabled())
+                throw ire(format("Cannot change transactional mode to %s for %s.%s with accord_transactions_enabled set to false",
+                                 next.transactionalMode, keyspaceName, tableName));
+
+            // user is manually updating migration mode, don't interfere
+            if (forceMigrationChange)
+            {
+                logger.warn("Forcing unsafe migration change from {} to {}", prev.transactionalMigrationFrom, next.transactionalMigrationFrom);
```

Comment:
Did we also log `transactionalMode` already? Can we include it in this statement?